app/src/main/res/values-vi/strings.xml

```diff
@@ -1,3 +1,251 @@
 <resources>
+    <string name="app_name">WSM</string>
     <string name="locate">vi</string>
+
+    <!-- Common -->
+    <string name="error">Lỗi</string>
+    <string name="retry">Thử lại</string>
+
+    <!-- Dialog -->
+    <string name="agree">Đồng ý</string>
+    <string name="disagree">Từ chối</string>
+    <string name="allow">Cho phép</string>
+    <string name="deny">Từ chối</string>
+    <string name="dont_ask_again">Không hỏi lại</string>
+    <string name="more_info">Thêm thông tin</string>
+    <string name="choose">Chọn</string>
+    <string name="clear">Xóa</string>
+
+    <!-- MenuDrawer -->
+    <string name="profile">HỒ SƠ CÁ NHÂN</string>
+    <string name="personal_information">Thông tin cá nhân</string>
+    <string name="setup_profile">Thiết lập cá nhân</string>
+    <string name="personal_data">DỮ LIỆU CÁ NHÂN</string>
+    <string name="working_calendar">Lịch làm việc</string>
+    <string name="holiday_calendar">Lịch ngày nghỉ</string>
+    <string name="personal_request">YÊU CẦU CÁ NHÂN</string>
+    <string name="request_overtime">Yêu cầu làm ngoài giờ</string>
+    <string name="request_day_off">Yêu cầu xin nghỉ</string>
+    <string name="request_come_late_leave_early">Yêu cầu đi trễ, về sớm</string>
+    <string name="workspaces_data">DỮ LIỆU WORKSPACES</string>
+    <string name="workspaces">Dữ liệu Workspaces</string>
+    <string name="statistics_of_personal">Thống kê cá nhân</string>
+    <string name="manage_requests">QUẢN LÝ YÊU CẦU</string>
+    <string name="manage_request_overtime">Quản lý yêu cầu làm ngoài giờ</string>
+    <string name="manage_request_off">Quản lý yêu cầu xin nghĩ</string>
+    <string name="manage_request_leave">Quản lý yêu cầu đi trễ, về sớm</string>
+
+    <string name="create_request_ot">Tạo yêu cầu làm ngoài giờ</string>
+    <string name="create_request_leave">Tạo yêu cầu đi trễ, về sớm</string>
+    <string name="create_request_off">Tạo yêu cầu xin nghỉ</string>
+    <string name="navigation_drawer_open">Mở ngăn điều hướng</string>
+    <string name="navigation_drawer_close">Đóng ngăn điều hướng</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.navigation_drawer_close` appears to be unused
 Unused resources make applications larger and slow down builds.